### PR TITLE
feat(lib): migrate to ng 10

### PR DESCRIPTION
**Pull request by danielsogl**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngneat/content-loader/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #56

## Does this PR introduce a breaking change?

```
[x] Yes
[ ] No
```

### CODE REVIEW

**Comment by NetanelBasal** [comment]
Why it's a draft?

**Comment by danielsogl** [comment]
@NetanelBasal I'd like to bump a few more deps but you can merge it if you want.

File: projects/ngneat/content-loader/tsconfig.lib.json [hunk 1]
```diff
@@ -12,21 +12,16 @@
...
   },
   "angularCompilerOptions": {
     "annotateForClosureCompiler": true,
     "skipTemplateCodegen": true,
     "strictMetadataEmit": true,
     "fullTemplateTypeCheck": true,
     "strictInjectionParameters": true,
-    "enableResourceInlining": true
+    "enableResourceInlining": true,
+    "enableIvy": false
```
Comment (danielsogl) [inline projects/ngneat/content-loader/tsconfig.lib.json:24]: Its recommended to disable ivy for libs

```bash
******************************************************************************
It is not recommended to publish Ivy libraries to NPM repositories.
Read more here: https://v9.angular.io/guide/ivy#maintaining-library-compatibility
******************************************************************************
```

**Comment by danielsogl** [comment]
@NetanelBasal I've finished the bumps

File: package.json [hunk 2]
```diff
@@ -13,51 +13,50 @@
     "contributors:generate": "all-contributors init"
   },
   "dependencies": {
-    "@angular/animations": "^8.2.6",
```
Comment (NetanelBasal) [inline package.json:16]: You need to add the --prod here:

    "build:lib": "ng build @ngneat/content-loader",

**Comment by NetanelBasal** [comment]
Add yourself to the contributors' list and verify that the build and the project work, please.

**Comment by danielsogl** [comment]
@NetanelBasal done

**Comment by NetanelBasal** [comment]
Verify that the build and the project work, please.

**Comment by danielsogl** [comment]
@NetanelBasal I did

**Comment by bjornharvold** [comment]
Can we see a release of this Angular 10 support commit please?

**Comment by NetanelBasal** [comment]
Yes

**Pull request merged**